shared/profile/user/weboftrust/index.tsx

```diff
@@ -81,13 +84,10 @@ const entry = (props: Props) => (
 const WebOfTrust = (props: Props) => {
   switch (props.status) {
     case WotStatusType.proposed: {
-      if (props.userIsYou) {
-        return entry(props)
-      }
-      return null
+      return entry(props, props.userIsYou ? 'Pending your approval:' : 'You proposed:')
```

Comment:
We should talk about this switch statement. Have some suggestions. Presently it's not very react-like and is not really necessary. It also bites you in the long-run.